src/transformers/training_args.py

```diff
@@ -1499,6 +1504,13 @@ class TrainingArguments:
         },
     )
 
+    include_num_input_image_tokens_seen: Optional[bool] = field(
```

Comment:
```suggestion
    include_num_input_image_tokens_seen: bool = field(
```

can you please also fix `include_num_input_tokens_seen`?: Optional[bool]` -> `include_num_input_tokens_seen: bool`

Reply:
Yes sure!